AllReadyApp/Web-App/AllReady/Services/TaskAttachmentService.cs

```diff
@@ -33,13 +33,13 @@ public AttachmentService(IOptions<AzureStorageSettings> options)
         /// <param name="organizationId">int ID</param>
         /// <param name="attachment">a attachment from Microsoft.AspNet.Http</param>
         /// <returns>URL to the uploaded file</returns>
-        public async Task<string> UploadTaskAttachmentAsync(int taskId, IFormFile attachment)
+        public async Task<string> UploadAsync(int taskId, IFormFile attachment)
         {
             var blobPath = "task/" + taskId.ToString();
```

Comment:
delete `.ToString()` here as the int is converted to a string by ways of string concatenation